x/group/testutil/app_config.go

```diff
@@ -119,5 +120,9 @@ var AppConfig = appconfig.Compose(&appv1alpha1.Config{
 				MaxMetadataLen:     255,
 			}),
 		},
+		{
+			Name:   minttypes.ModuleName,
+			Config: appconfig.WrapAny(&mintmodulev1.Module{}),
+		},
```

Comment:
This seems unreleated to this PR

Reply:
this is added to fix the group module test because this pr changes the `InitGenesis` behavior: before this pr empty genesis data of a specific module leads to not calling {module}.`InitGenesis`, but now {module}.`InitGenesis` would be called anyway

Reply:
Why do we need to do this? Seems unrelated...

Reply:
```golang
func (m *Manager) InitGenesis(ctx sdk.Context, cdc codec.JSONCodec, genesisData map[string]json.RawMessage) abci.ResponseInitChain {
	var validatorUpdates []abci.ValidatorUpdate
	ctx.Logger().Info("initializing blockchain state from genesis.json")
	for _, moduleName := range m.OrderInitGenesis {
		ctx.Logger().Debug("running initialization for module", "module", moduleName)
                 // if genesisData[moduleName] == nil { continue }
		moduleValUpdates := m.Modules[moduleName].InitGenesis(ctx, cdc, genesisData[moduleName])
```
I changed the `initGenesis` behavior so that I have to initialize the `mint` module to avoid the group module test panics for `m.Modules["mint"]` being nil pointer:
https://github.com/cosmos/cosmos-sdk/blob/main/x/group/module/abci_test.go#L42